k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeMessage.java

```diff
@@ -48,6 +48,9 @@
     protected Address[] mCc;
     protected Address[] mBcc;
     protected Address[] mReplyTo;
+    protected Address[] mX_OriginalTo;
```

Comment:
We're moving away from using the `m` prefix for fields. Please don't use it for new fields.
Also, please don't use underscores in field names. This should be

```java
protected Address[] xOriginalTo;
protected Address[] deliveredTo;
protected Address[] xEnvelopeTo;
```